python/sqlflow_submitter/xgboost/pai_rabit.py

```diff
@@ -64,8 +64,7 @@ def __init__(self, host, nworkers, port):
                                 self.TrackerHelperHTTPRequestHandler)
         tracker_port = PaiXGBoostTracker.find_free_port()
         nslave = nworkers
-        tracker = xgb.RabitTracker(host, nslave, tracker_port,
-                                   tracker_port + 1)
+        tracker = RabitTracker(host, nslave, tracker_port, tracker_port + 1)
```

Comment:
Why still need to copy the `tracker.py` file? For PAI XGBoost install version reasons?

Reply:
Yes.
I added a comment [here](https://github.com/sql-machine-learning/sqlflow/pull/2134#discussion_r422162527). Let's remove `tracker.py` when PAI tf115 supports Python3.